tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "jsx": "react",
```

Comment:
WAT ! is this a gitmoji ?

Reply:
Alright I put back the wrench 🚫💪

Reply:
I was just joking to say "I saw it" and didn't dislike it #litote